src/components/VersionSelector/VersionSelector.tsx

```diff
@@ -21,16 +21,32 @@ import { useOutsideClick } from './use-outside-click';
 const VersionSelectorComponent = ({
   resourceVersions,
   active,
-  rootUrl,
   description,
+  rootUrl,
 }: VersionSelectorProps): JSX.Element => {
-  const [selectedIdx, setSelectedIdx] = React.useState(
-    resourceVersions.indexOf(active.resourceVersion),
-  );
+  const initialSelectedIdx = resourceVersions.indexOf(active.resourceVersion);
```

Comment:
Are we changing this from the index of the resourceVersion to the string of the resourceVersion?

Reply:
Oh, I think this was caught up in resolving merge conflicts. I think the `initialSelectedState` is just used to name the value for the `  const [selectedIdx, setSelectedIdx] = React.useState(initialSelectedIdx);` on line 30. I can change it back though if you would like!

Reply:
Nope, you're right!! This was just confusing in github!